deploy/rp-production-parameters.json

```diff
@@ -47,6 +47,9 @@
         "databaseAccountName": {
             "value": ""
         },
+        "dbtokenClientId": {
```

Comment:
Do we need to update the prod config to include a value for this?

Reply:
Yes.  I need elevated JIT when this is merged in to create these.